code/datums/elements/noticable_organ.dm

```diff
@@ -53,16 +51,19 @@
 
 	if(!should_show_text(examined))
 		return
-	var/examine_text = replacetext(replacetext("[body_zone ? examined.p_Their() : examined.p_They()] [infused_desc]", "%PRONOUN_ES", examined.p_es()), "%PRONOUN_S", examined.p_s())
-	if(add_span)
-		examine_text = span_notice(examine_text)
+
+	var/examine_text = infused_desc
+	/// Helper proc to iterate over whatever message and replace
+	/// pronouns where necessary
+	examine_text = REPLACE_PRONOUNS(examine_text, examined)
```

Comment:
```suggestion
	var/examine_text = REPLACE_PRONOUNS(infused_desc, examined)
```
1. Comment isn't super necessary, one could reasonable infer this is what's going on (or check documentation for `REPLACE_PRONOUNS`
2. No need to set examine text to infused_desc if we're immediately going to replace it